packages/react/src/generators/application/__snapshots__/application.spec.ts.snap

```diff
@@ -222,8 +222,8 @@ if (import.meta.vitest) {
   });
 
   it('should have a greeting as the title', () => {
-    const { getByText } = render(<App />);
-    expect(getByText(new RegExp('Welcome plain', 'gi'))).toBeTruthy();
+    const { getAllByText } = render(<App />);
```

Comment:
How come this needed to change?

Reply:
React Testing Library is returning multiple elements containing the text

Reply:
only one exists, but the parents seem to be included rather than the last child